docs/reference/team-processes/release-process.md

```diff
@@ -2,146 +2,177 @@
+
 If you are the designated release owner, consider the following suggestions to managing your work and the release:
 
 * When taking on work for the sprint, consider taking on around 5-15% less in order to allocate a portion of each day to release management duties and in particular towards the end of the sprint - this helps give you padding in case an unexpected (or two, or three) dot release comes up.
 * Take some time a day or two before tagging is scheduled to reach out in the #fxa-team Slack channel or to individual team members to check-in on in-progress or to-do work items. This might result in you taking on additional work performing reviews or reaching out to the appropriate people.
     * Things to consider asking:
         * Are there any tasks that need to be in the train? Are any of them in jeopardy of holding up the release from being tagged?
         * Is there anything I can do to help keep things moving?
-* 1-3 hours before tagging, check open PRs and let the team know you will be tagging at X time. Atypically, a critical patch may be very close to landing which may delay the tag by a few hours.
+* 1-3 hours before tagging, check open PRs and let the team know you will be tagging at X time and ask that any missing notes in the deploy doc be added. Atypically, a critical patch may be very close to landing which may delay the tag by a few hours.
 
 :::tip
-If you're new to release ownership, check that you have access to the [fxa-private](https://github.com/mozilla/fxa-private/) and [cloudops-deployment](https://github.com/mozilla-services/cloudops-deployment) repos. While not crucial, access to fxa-private enables you to see a Bugzilla URL outputted by the release script and access to cloudops-deployment allows you to see PRs that may be listed in the deploy doc as well as make PRs in that repo when you make code changes that necessitate them.
+If you're new to release ownership, check that you have access to the [`fxa-private`][fxa-private] and [`cloudops-deployment`][cloudops-deployment] repos. While not crucial, access to fxa-private enables you to see a Bugzilla URL outputted by the release script and access to cloudops-deployment allows you to see PRs that may be listed in the deploy doc as well as make PRs in that repo when you make code changes that necessitate them.
 :::
 
 ## Releasing Code
 
-1. Release ownership is confirmed during the sprint kick-off team meeting.
+:::caution
+**The release script expects the git origin to be unchanged from the default.**  If you've modified your git remotes you will get confusing output here and might mess things up.  If in doubt, check out a new copy of FxA (eg. `git clone git@github.com:mozilla/fxa.git fxa.tagging` and do all your tagging there.
+
+**If you're tagging in a newly cloned repo, ensure your commits will be GPG signed.** Run `git config --list` and verify you see `commit.gpgsign=true`. If this is not already set globally, run `git config --global commit.gpgsign true`.
```

Comment:
Might be worth linking to a GitHub docs page on this, as the initial set up can be involved IIRC.